pom.xml

```diff
@@ -44,8 +44,8 @@ THE SOFTWARE.
 
   <properties>
     <jenkins.version>1.580.1</jenkins.version>
-    <java.level>6</java.level>
-    <mavenInterceptorsVersion>1.7</mavenInterceptorsVersion>  
+    <java.level>7</java.level>
```

Comment:
I know why you are doing this for the maven compatibility for 3.3 which requires j7 but in that case we should update the minimal jenkins version to Jenkins OSS >= 1.612 or Jenkins LTS >= 1.625.1
We should avoid older versions of jenkins which may run on J6 use this version
In that case I think that we should release a new version of the plugin before integrating this change